geometry_msgs/PointStamped[] basket_locs
---
string[] basket_colours

